### DISCUSSION TITLE
Change font color of Qselect

### DISCUSSION THREAD

**Discussion by manimovassagh** [quasarframework/quasar#14133] [General - Components / Directives / etc] [answered]
Hi Community,
i have a qselect item as follows
https://codepen.io/manimovassagh/pen/abYYpPr?editors=1010
i want to change default font color (Black default) to another one .
the hover blue color is ok , i want to only change the default font colot from black to something like grey.
but it is not working i tried diffrent way like inline styling and another things but could not figure out.
also another question : is there any way that i use as div slots inside and can change the style?
something like this:
 <q-select borderless v-model="model"  >
bring options as div slots here and then manipulate styling?
</q-select>
Thanks a lot
Mani

**Comment by MilosPaunovic** [comment, marked as answer]
Use `popup-content-class` or `popup-content-style` props.
https://codepen.io/milospaunovic/pen/OJvvrLg?editors=1010

**Comment by manimovassagh** [comment]
Thanka a lot .helped me really well. i just note that also it is not working with scoped css but for me it worked for now.
Thanks a lot
